spyder/plugins/onlinehelp/widgets.py

```diff
@@ -71,7 +71,8 @@ def spyder_safeimport(path, forceload=0, cache={}):
 
 # Needed to prevent showing a warning message regarding debugging
 # See spyder-ide/spyder#20390
-if is_pynsist():
+# ??? Do we need this?
```

Comment:
I'll check it.

Reply:
I'm getting the error `The baseline image directory does not exist. This is most likely because the test data is not installed. You may need to install matplotlib from source to get the test data.` when searching the online help from the master branch, prior to the changes in this PR.